Makefile

```diff
@@ -31,7 +31,7 @@ export PRINT_HELP_PYSCRIPT
 
 BROWSER := python3 -c "$$BROWSER_PYSCRIPT"
 
-DOCKER_IMAGE?=us.gcr.io/vcm-ml/fv3gfs-wrapper:gnu9-mpich314-nocuda
+DOCKER_IMAGE?=us.gcr.io/vcm-ml/fv3gfs-wrapper:gnu7-mpich314-nocuda
```

Comment:
```suggestion
# This must be updated if the tag used in docker/Makefile is changed
DOCKER_IMAGE?=us.gcr.io/vcm-ml/fv3gfs-wrapper:gnu7-mpich314-nocuda
```

This warning comment should be enough for this PR, I'm still wondering how we can run the GPU tests with the makefile in this state.